### ISSUE TITLE
Library update breaks app on up to date pop os

### ISSUE THREAD

**Issue by ragardner** [ProtonVPN/proton-vpn-gtk-app#13]
>  We are happy to answer your questions about the code or discuss technical ideas. 
 
Please complete the following checklist (by adding [x]):
- [x] I have searched open and closed issues for duplicates
- [x] This isn't a feature request
- [x] This is not a report about my app not working as expected

----------------------------------------

Following an update to proton vpn libraries on 27th nov, today the app disappeared from the "Pop!_Shop" app center and dock

I uninstalled using instructions on your website help and tried to reinstall but I am getting this message:

```
The following packages have unmet dependencies.
 python3-proton-vpn-session : Breaks: python3-proton-vpn-api-core (< 0.20.2) but 0.20.1 is to be installed
E: Unable to correct problems, you have held broken packages.
```

**Comment by hasan515pr** [comment]
I have same problem and i posted it, anyone can help?

**Comment by KissesRefined** [comment]
I noticed a similar update issue today

dnf error messages:
```
$ sudo dnf --refresh upgrade
...
Problem: package python3-proton-vpn-session-0.6.3-1.fc39.noarch from protonvpn-fedora-stable conflicts with python3-proton-vpn-api-core < 0.20.2 provided by python3-proton-vpn-api-core-0.20.1-1.fc39.noarch from @System
  - cannot install the best update candidate for package python3-proton-vpn-session-0.6.2-1.fc39.noarch
  - cannot install the best update candidate for package python3-proton-vpn-api-core-0.20.1-1.fc39.noarch
==============================================================================================================================================================
 Package                                        Architecture               Version                          Repository                                   Size
==============================================================================================================================================================
Skipping packages with conflicts:
(add '--best --allowerasing' to command line to force their upgrade):
 python3-proton-vpn-session                     noarch                     0.6.3-1.fc39                     protonvpn-fedora-stable                      76 k

Transaction Summary
==============================================================================================================================================================
Skip  1 Package
```
System and package info:
```
uname -a 
Linux hws 6.5.12-300.fc39.x86_64 #1 SMP PREEMPT_DYNAMIC Mon Nov 20 22:44:24 UTC 2023 x86_64 GNU/Linux

cat /etc/fedora-release 
Fedora release 39 (Thirty Nine)

$ dnf list --installed | grep proton
proton-vpn-gnome-desktop.noarch                      0.2.0-1.fc39                        @protonvpn-fedora-stable
proton-vpn-gtk-app.noarch                            4.1.0-1.fc39                        @protonvpn-fedora-stable
protonvpn-stable-release.noarch                      1.0.1-2                             @@commandline           
python3-proton-core.noarch                           0.1.15-1.fc39                       @protonvpn-fedora-stable
python3-proton-keyring-linux.noarch                  0.0.1-1.fc39                        @protonvpn-fedora-stable
python3-proton-keyring-linux-secretservice.noarch    0.0.1-1.fc39                        @protonvpn-fedora-stable
python3-proton-vpn-api-core.noarch                   0.20.1-1.fc39                       @protonvpn-fedora-stable
python3-proton-vpn-connection.noarch                 0.11.0-1.fc39                       @protonvpn-fedora-stable
python3-proton-vpn-killswitch.noarch                 0.2.0-1.fc39                        @protonvpn-fedora-stable
python3-proton-vpn-killswitch-network-manager.noarch 0.2.0-1.fc39                        @protonvpn-fedora-stable
python3-proton-vpn-logger.noarch                     0.2.1-1.fc39                        @protonvpn-fedora-stable
python3-proton-vpn-network-manager.noarch            0.3.0-1.fc39                        @protonvpn-fedora-stable
python3-proton-vpn-network-manager-openvpn.noarch    0.0.4-1.fc39                        @protonvpn-fedora-stable
python3-proton-vpn-session.noarch                    0.6.2-1.fc39                        @protonvpn-fedora-stable
```

**Comment by calexandru2018** [comment]
Updates have been pushed. We had some internal issues with releases which caused the breaking package to not be updated. We've since fixed that and all packages are on our repos, just refresh the repos.

Closes duplicates #15 #14
